### ISSUE TITLE
Provide a Tutorial Guide showing how to "ping a collector install" with a dummy signal (e.g., metric) without the use of a language SDK or other component

### ISSUE THREAD

**Issue by genemat** [open-telemetry/opentelemetry.io#729]
The current Getting Started guide includes a demo that use Zipkin, Jaeger and Prometheus. I am looking for a more simpler guide. Something that would show how to send a metric(s) using a utility like CURL and seeing the metric being recorded in a local file. This would help confirm that the installation of OTEL Collector is working normally.

**Comment by austinlparker** [comment]
I think something like https://github.com/equinix-labs/otel-cli might be useful for this under the collector quick start as it can send gRPC (which is, presumably, what most people using a collector + OTLP will be using).

**Comment by genemat** [comment]
Yes otel-cli looks like a great and capable OTEL utility to quickly and simple send metric data to OTEL, however, I'm looking something even more simpler, without GO and Jaeger. I am looking to confirm that OTEL is installed and working correctly.

Is there a curl command that can be used, with dummy random metric data, to send to the OTEL collector listener and confirm the receipt of data to a local file? This will confirm that OTEL is up and running and accepting connections.

**Comment by austinlparker** [comment]
I'm not sure if the metric receiver supports measurements in formats other than gRPC at this point? Might be a better question for the collector repository. You can confirm receipt of data pretty easily with the logging exporter on the collector though.

**Comment by genemat** [comment]
Thank You! I'm just starting my journey with Opentelemetry . I am wondering if a simple curl command can be formatted using Opencensus format to send a dummy metric to OTEL? If there was a guide on how to do this it would be immensely helpful.

**Comment by cartermp** [comment]
A bit wordy, but I renamed this issue to be a little more clear about the intent. We do offer what you're asking for in language docs today (e.g., the python docs show how to set up a local collector and send data to it), but there's nothing that's purely collector-focused and doesn't involve any other components.

**Comment by mc-chaos** [comment]
Hi @cartermp ,
we use [telemetrygen](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/cmd/telemetrygen) from opentelemetry collector contrib Repo.
It works for us.
Regards, Sascha

**Comment by svrnm** [comment]
@theletterf This may be addressed with an updated getting started guide for the collector as well (https://github.com/open-telemetry/opentelemetry.io/issues/1695)

**Comment by svrnm** [comment]
fixed via https://github.com/open-telemetry/opentelemetry.io/issues/1695
